docs/content/python/guides/roles/sqlalchemy/getting-started.md

```diff
@@ -10,7 +10,7 @@ description: >
 When managing access to resources within an application, it can be useful to
 group permissions into **roles**, and assign these roles to users. This is
 known as **Role-Based Access Control (RBAC).** The SQLAlchemy roles
-library extends the `Oso` core library with built in configuration,
+library extends the `Oso` core library with built-in configuration,
```

Comment:
It's really extending `sqlalchemy-oso` more than `oso` — if anything, the relationship between the SQLAlchemy roles library & the `oso` Python library is one of shadowing